types/bootstrap/js/dist/alert.d.ts

```diff
@@ -0,0 +1,38 @@
+declare class Alert {
+    constructor(element: Element);
+
+    /**
+     * Closes an alert by removing it from the DOM. If the .fade and .show
+     * classes are present on the element, the alert will fade out before it
+     * is removed.
+     */
+    close: () => void;
+
+    /**
+     * Destroys an element's alert.
+     */
+    dispose: () => void;
+}
+
+declare namespace Alert {
+    enum Events {
+        /**
+         * Fires immediately when the close instance method is called.
+         */
+        close = 'close.bs.alert',
+
+        /**
+         * Fired when the alert has been closed and CSS transitions have
+         * completed.
+         */
+        closed = 'closed.bs.alert',
+    }
+
+    /**
+     * Static method which allows you to get the alert instance associated to a
+     * DOM element, you can use it like this: getInstance(alert)
+     */
+    function getInstance(element: Element): Alert;
```

Comment:
If this is a static method, why not declare it as a static method on the `Alert` class?

Reply:
good point. Thank you